panel/pane/plotly.py

```diff
@@ -311,13 +310,12 @@ def _process_param_change(self, params):
         return props
 
     def _get_model(
-        self, doc: Document, root: Optional[Model] = None,
-        parent: Optional[Model] = None, comm: Optional[Comm] = None
+        self, doc: Document, root: Model | None = None,
+        parent: Model | None = None, comm: Comm | None = None
     ) -> Model:
-        if not hasattr(self, '_bokeh_model'):
-            self._bokeh_model = lazy_load(
-                'panel.models.plotly', 'PlotlyPlot', isinstance(comm, JupyterComm), root
-            )
+        Plotly._bokeh_model = lazy_load(
```

Comment:
Wouldn't this be overwritten each time `_get_model` is called? 

Not that well versed with the internal logic here, but I would assume it should still be done behind an if statement?

Reply:
It's "overwritten" with the same value, and `lazy_load` has side-effects that MUST run. So this is correct.